### DISCUSSION TITLE
cask for vscodium-insiders

### DISCUSSION THREAD

**Discussion by GitMensch** [Homebrew/discussions#4815] [Casks] [answered]
### Output of `brew config`

```shell
nA
```

### Output of `brew doctor`

```shell
nA
```

### Description of issue

It seems reasonable to copy the existing cask https://github.com/Homebrew/homebrew-cask/blob/master/Casks/v/vscodium.rb creating a vscodium-insiders one only changing the name/description and pointing the download to https://github.com/VSCodium/vscodium-insiders - but as I don't actually use brew I can't say if/how the resulting installation works, so I don't think I should produce that myself...

### Relevant casks

vscodium

**Comment by miccal** [comment, marked as answer]
https://github.com/Homebrew/homebrew-cask-versions/blob/HEAD/Casks/vscodium-insiders.rb

**Reply by GitMensch** [reply]
I've missed that:

> brew tap homebrew/cask-versions

Thanks for pointing it out!
